### DISCUSSION TITLE
Student pack requesting full name in github username

### DISCUSSION THREAD

**Discussion by ghost** [community/community#57851] [GitHub Education]
### Hi everyone,

Hello!

Upon applying for the GitHub student package, a few days passed and I received the message that I must change my GitHub username to my real life name, which is something I am really not comfortable with, what should I do?

![Screenshot 2023-06-12 202015](https://github.com/community/community/assets/67878277/841c7f6e-930c-4c32-9c16-59302852da2f)

thanks

**Comment by mrepol742** [comment]
You only need to update your github profile name to your real world name. Since you are not comfortable sharing your name.
PS: i dont know if this may work 

You can change your name to real name then apply for github student pack and then change it later on (after they approved your request).

**Reply by P10swati** [reply]
Hey could you apply for it ?

**Reply by mrepol742** [reply]
Here
https://education.github.com/

**Reply by ghost** [reply]
But that's an overly long time, and since my Github account is kinda linked to other profiles, it's likely that someone may be able to see it since for me it took like 9 days, is there any other solution or should I give up on this?

**Comment by FlawlessDeveloper** [comment]
I also have the same issue. That was a, in my opinion a very unnecessary change to the verification process.

**Reply by cosmos5555** [reply]
Hey quick question. That's a cool profile picture where did you get it from and when was it made?

**Reply by FlawlessDeveloper** [reply]
> Hey quick question. That's a cool profile picture where did you get it from and when was it made?

@DigitalCosmos555 It was made using pixel sorting by a friend of mine a few years back. I cant really remember the source image tho.

**Reply by cosmos5555** [reply]
Haha, no way, that is actually really funny. I have to tell you—your friend's art is currently at the center of a massive, super weird internet conspiracy theory right now.

I don't know if you follow online rabbit holes, but there's a viral theory going around about a 'time traveler' Twitter account, and they used your exact profile picture. People realized that if you superimpose the famous photo of the 2024 Trump assassination attempt in Butler over it, the colors and shapes match up pretty close. Now a bunch of people are theorizing that your picture is actually the Trump photo sent back in time and pixel-sorted by a time traveler.

Looking closely at it though, it would obviously have to be something more than just standard pixel sorting for that conspiracy to be true. The blue sky in the Trump photo is a fairly solid color, so a standard sort wouldn't generate so many different colors at the top of the image.

I was investigating the origin, which is how I stumbled onto your GitHub. I noticed per the Wayback Machine that you’ve had this PFP since at least April 2022. That actually supports their timeline since it predates the 2024 event, but obviously it doesn't prove it outright.

It would be really interesting to see what the real source image was. Is there any chance you could hit up your friend to see if they still have the original unedited file, or remember what specific script/app they used? It would be cool to see what the base image actually was---like if it was originally just a regular landscape photo, a cityscape, or something totally random—so we can see where this rabbit hole leads.

**Reply by cosmos5555** [reply]
https://youtu.be/Q6Q7qgEAmTc ah here is some more background if your interested in looking at it more 😂

**Comment by amirroox** [comment]
You can change the name only to receive the package.
and choose a nickname after receiving it.
Also, if you have a problem with the real name, you can send a message to GitHub users to check your verification.

**Reply by FlawlessDeveloper** [reply]
What way is the correct way to address that? Where can i contact them?

**Reply by amirroox** [reply]
You can use the link below:

**https://support.github.com/features/education**
**https://support.github.com/tickets/personal/0**

**Reply by FlawlessDeveloper** [reply]
I created a ticket there at wednesday. How long is the average wait time? Can i somehow accelerate that? Thank you for your help already 😄

**Reply by amirroox** [reply]
It may take up to a week. Unfortunately, I don't think there is a way to speed it up 💔

**Reply by FlawlessDeveloper** [reply]
Thank you for your help anyway. I wish you a great day/night

**Reply by Mirakud** [reply]
Any update ? @FlawlessDeveloper

**Comment by masukomi** [comment]
to the people suggesting you can change your name only for a short time "up to a week" I think you are failing to recognize the issue here.

Once someone has exposed their name, it's exposed. Any party that they didn't want to see it has had up to a week to see it. 

Imagine, for example, you're a woman in a community where it's acceptable to violently beat women for trying to educate themselves. Would you say that forcing those people to expose themselves is reasonable?  Would it be reasonable to ask people being persecuted by governments for daring to speak out about them to expose themselves publicly? What about women who have had to hide themselves from stalkers who ignore restraining orders?

Real name verification _may_ be required to verify academic affiliation but PUBLIC exposure of that name is not.

**Reply by fernvenue** [reply]
Totally agree, and since we have Web Archive now, so we don't know is it really just a week or not.

**Reply by diminDDL** [reply]
You are correct, this is an extreme violation of ones desire to keep their online and irl lives separate and there is absolutely no reason for GitHub to require you to expose this information publicly. It's also concerning that they didn't respond to this question in any meaningful way.

**Comment by FulytheFox** [comment]
Since I'm a student at a school participating in the GitHub Education program since this year, I wanted to apply there and got exactly the same issue.

As the op wrote, I am uncomfortable exposing my full name **publicly** on my profile (even its only for a few days). This is an extreme violation of privacy rights and I can't understand why this is a requirement for the application process. There would be several other solutions GitHub could've made the verification of the name more privacy friendly like:

- an extra input field for the full name under the document upload in the application process
- an option to make the profile name private
- matching the school mail name
- the possibility to upload a picture of an ID card or passport

**Comment by velecto1** [comment]
Well... I did what the form asked me for, including logging in and logging out, and it still says the same!

**Comment by PumpkinXD** [comment]
I've given up since I can't hide my real name while applying the pack

kinda... dumb

**Reply by ghost** [reply]
Yeah I've given up a long time ago.
